Library/Homebrew/rubocops/lines.rb

```diff
@@ -464,38 +464,45 @@ class GenerateCompletionsDSL < FormulaCop
         def audit_formula(_node, _class_node, _parent_class_node, body_node)
           install = find_method_def(body_node, :install)
 
-          correctable_shell_completion_node(install) do |node, shell, base_name, binary, cmd, shell_parameter|
-            next unless shell_parameter.match?(/(bash|zsh|fish)/) # generate_completions only applicable if shell is passed
+          correctable_shell_completion_node(install) do |node, shell, base_name, executable, cmd, shell_parameter|
+            # generate_completions_from_executable only applicable if shell is passed
+            next unless shell_parameter.match?(/(bash|zsh|fish)/)
 
             base_name = base_name.delete_prefix("_").delete_suffix(".fish")
-            shell = shell.to_s.sub("_completion", "").to_sym
-            binary = binary.source
+            shell = shell.to_s.delete_suffix("_completion").to_sym
+            executable = executable.source
             shell_parameter_stripped = shell_parameter.sub("bash", "").sub("zsh", "").sub("fish", "")
-            shell_prefix = if shell_parameter_stripped.empty?
+            shell_parameter_format = if shell_parameter_stripped.empty?
               nil
             elsif shell_parameter_stripped == "--"
-              :flag
+              ":flag"
+            elsif shell_parameter_stripped == "--shell="
+              ":arg"
```

Comment:
FYI you could probably just leave this as a symbol and then use `#inspect` later to put the `:` and/or `"`